lib/mayaUsd/nodes/proxyShapePlugin.cpp

```diff
@@ -62,6 +63,9 @@ MayaUsdProxyShapePlugin::initialize(MFnPlugin& plugin)
         return MS::kSuccess;
     }
 
+    // Call one time registration of plugins compiled for same USD version as MayaUSD plugin.
+    MAYAUSD_NS::registerVersionedPlugins();
```

Comment:
Here in the core, you should just use MayaUsd::

Reply:
As per offline discussion, I removed all usages of MAYAUSD_NS.